lib/server.js

```diff
@@ -0,0 +1,115 @@
+AntiSearchSource.allow = function(collectionName, allowRules) {
+  check(allowRules, {
+    maxLimit: Match.Optional(Number),
+    securityCheck: Match.Optional(Function),
+    allowedFields: Match.Optional([
+      Match.OneOf(String, {
+        collection: String,
+        referenceField: String,
+        fields: [String]
+      })
+    ]),
+    queryTransform: Match.Optional(Function)
+  });
+  this._allowRules[collectionName] = allowRules;
+  return this._allowRules[collectionName];
+};
+
+AntiSearchSource._allowRules = {};
+
+Meteor.methods({
+  __makeGlobalSearch: function(configEntry) {
+
+    let collection, currentAllowRule, fieldsToPublish, queries, searchQuery, userId;
```

Comment:
@IuriiBod write it while using let doesn't make sense. let's scoping works fine and is obvious.